tasks/test-metadata/0.2/test-metadata.yaml

```diff
@@ -106,3 +114,18 @@ spec:
 
         # Write job-spec to result file
         echo -n "$JOB_SPEC" > $(results.job-spec.path)
+
+        if [ "$ENABLE_PREFETCH" == "true" ]; then
+
+          cosign download attestation "$COMPONENT_CONTAINER_IMAGE" > cosign_metadata.json
+
+          CACHI2_SOURCE_ARTIFACT="$(jq -r \
+            '.payload | @base64d | fromjson | .predicate.buildConfig.tasks[] |
+            select(.name == "prefetch-dependencies") | .results[] | select(.name == "CACHI2_ARTIFACT") | .value' \
+            cosign_metadata.json)"
```

Comment:
@ccronca was this tested against some specific container image produced by konflux? if so, could you share it